src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,95 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "validLogin";
+    private static final String SHORT_LOGIN = "log";
+    private static final String VALID_PASSWORD = "validPassword";
+    private static final String SHORT_PASSWORD = "pass";
+    private static final int VALID_AGE = 22;
+    private static final int YOUNGER_AGE = 17;
+    private static final int NEGATIVE_AGE = -20;
+    private static final String NULL_VALUE = null;
+    private static RegistrationService registrationService;
+    private final User user = new User();
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        user.setLogin(VALID_LOGIN);
+        user.setPassword(VALID_PASSWORD);
+        user.setAge(VALID_AGE);
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.PEOPLE.clear();
+    }
+
+    @Test
+    void register_userIsValid_Ok() throws RegistrationException {
```

Comment:
Why do you need throws unchecked exception? We're not supposed to do that and anyway, in which method do you throw it?
```suggestion
    void register_userIsValid_Ok() {
```